libs/chain-signatures/contract/src/state/running.rs

```diff
@@ -294,17 +301,12 @@ pub mod running_tests {
 
         let original_epoch_id = state.keyset.epoch_id;
         let mut resharing = None;
-        for (i, (account_id, _, _)) in participants
-            .participants()
-            .iter()
-            .enumerate()
-            .take(state.parameters.threshold().value() as usize)
-        {
+        for (i, (account_id, _, _)) in proposal.participants().participants().iter().enumerate() {
             env.set_signer(account_id);
             let res = state
                 .vote_new_parameters(state.keyset.epoch_id.next(), &proposal)
                 .unwrap();
-            if i + 1 < state.parameters.threshold().value() as usize || num_domains == 0 {
+            if i + 1 < proposal.participants().len() || num_domains == 0 {
```

Comment:
Ditto on binding bools to descriptive variable names.

Reply:
I would suggest just commenting.
```
// If we don't have all votes, or if we have no domains, then we shouldn't be transitioning state.
```
(What would the variable be named otherwise?)

Reply:
I see now that this is test code, but in general if you have descriptive variable names you can avoid comments that can eventually become stale, and still keep it easy to follow.

```suggestion
            let missing_threshold_number_of_votes =  i + 1 < proposal.participants().len();
            let has_no_domains = num_domains == 0;
            if missing_threshold_number_of_votes || has_no_domains {
```